Source/WebCore/rendering/RenderLayer.h

```diff
@@ -1380,10 +1388,8 @@ class RenderLayer final : public CanMakeSingleThreadWeakPtr<RenderLayer>, public
     RenderLayer* m_last { nullptr };
 
     SingleThreadWeakPtr<RenderLayer> m_backingProviderLayer;
-
-#if ASSERT_ENABLED || ENABLE(CONJECTURE_ASSERT)
-    SingleThreadWeakPtr<RenderObject> m_repaintContainer;
-#endif
+    SingleThreadWeakPtr<RenderLayer> m_backingProviderLayerAtEndOfCompositingUpdate;
```

Comment:
Is this used outside of assertions? If not, #ifdef it.

Reply:
It's used as part of the actual logic (which I am slightly sad about).